lib/rspec/core/formatters/html_formatter.rb

```diff
@@ -69,10 +69,11 @@ def example_failed(failure)
           example = failure.example
 
           exception = failure.exception
+          message_lines = failure.fully_formatted_lines(nil, RSpec::Core::Notifications::NullColorizer)
           exception_details = if exception
                                 {
-                                  :message => failure.message_lines.join("\n"),
-                                  :backtrace => failure.formatted_backtrace.join("\n")
+                                  :message => message_lines.drop(2).join("\n"),
+                                  :backtrace => failure.formatted_backtrace.join("\n"),
```

Comment:
Funny, I thought Ruby 1.8 didn't support a trailing comma on the last collection element.  Nice to see that it does!